client/index.jsx

```diff
@@ -30,10 +38,35 @@ class App extends React.Component {
     this.getReviews();
   }
 
+  showThreeReviews() {
+    this.setState({
+      threeReviews: !this.state.threeReviews
+    })
+  }
+
+  showAllReviews() {
+    this.setState({
+      fullReviews: !this.state.fullReviews
+    })
+  }
+
   render() {
+
     return (
       <div>
-        <h1> Hello test test </h1>
+        {this.state.fullReviews ? <FullReviews showAllReviews={this.showAllReviews.bind(this)} numStars={this.state.avgStars} reviews={this.state.reviews}/>: null}
+        <div id="reviews-header" onClick={() => this.showThreeReviews()}>
+          <span>Reviews ({this.state.reviews.length})</span>
+          <div>
+            <span className='stars'>
+              <Stars numStars={this.state.avgStars}/>
+            </span>
+            <span>
+              {this.state.threeReviews ? <i className="arrow up"></i>: <i className="arrow down"></i>}
+            </span>
+          </div>
+        </div>
+        {this.state.threeReviews ? <SummaryReviews numStars={this.state.avgStars} reviews={this.state.reviews.slice(0,3)} showAllReviews={this.showAllReviews.bind(this)}/>: null}
       </div>
```

Comment:
Great architectural structure!
Overall your react components are very clear and understandable as well as the structure.
I don't have any feedback to provide as far as necessary changes, etc.